transformer_engine/pytorch/distributed.py

```diff
@@ -856,3 +865,110 @@ def allreduce(
     handle = torch.distributed.all_reduce(input_, group=tp_group, async_op=async_op)
 
     return input_, handle
+
+
+def _fsdp_scatter_tensors(
```

Comment:
Interesting that the linter doesn't complain about the missing docstring, is this due to the `__all__` decl or just using internal function convention with `_func_name`? Either way I think this is good practice going forward as well instead of adding filler docs!

Reply:
Yes, I believe PyLint ignores some warnings by default for internal functions designated with the leading underscore.